cli/user/README.md

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega User CLI
+
+Pravega User CLI is a tool for testing Pravega deployments (it is based on the 
+former [Interactive Demo Tool](https://github.com/pravega/pravega/wiki/Interactive-Demo-Tool)).
+
+## Prerequisites
+
+- Java 8+
+
+## Building Pravega User CLI
+
+Checkout the source code:
+
+```
+git clone https://github.com/pravega/pravega
+cd pravega
+```
+
+Build the Pravega User CLI tool:
+
+```
+./gradlew distTar
+```
+
+Unzip the result distribution artifact:
+
+```
+cd build/distributions/
+tar -xzvf pravega-xxx-SNAPSHOT.tar
+```
+
+You will find the executable file (`pravega-cli`) as well as the default configuration under the
+`bin` directory.
+
+# Executing the Pravega User CLI
+
+Next, we show how to use the Pravega User CLI tool.
+
+> Before using the Pravega User CLI, we first need a Pravega cluster up and running. While the simplest way is the 
+[standalone deployment](http://pravega.io/docs/latest/deployment/run-local/), you can explore other ways 
+of deploying Pravega in the project's [documentation](http://pravega.io/docs/latest/deployment/deployment/). 
+
+You can run the Pravega User CLI as follows:
+```
+./bin/pravega-cli
+```
+You will se an output related to the default configuration parameters:
+```
+Pravega User CLI Tool.
+	Usage instructions: https://github.com/pravega/pravega/wiki/Interactive-Demo-Tool
```

Comment:
Since this page is empty, let's rename it to something more appropriate and update code references to it.

Now would be the best time, before we start advertising and filling it.

Reply:
Agree, I have renamed the page as follows: https://github.com/pravega/pravega/wiki/Pravega-User-CLI. I have also replaced all the occurrences in the code with the new link.